docs/security.md

```diff
@@ -750,3 +750,18 @@ Use the following option to enable this feature:
 | YAML                 | `EnableTrustedAuth`              |
 | Environment variable | `EVENTSTORE_ENABLE_TRUSTED_AUTH` |
 
+## FIPS 140-2
+
```

Comment:
I'd make a note directly along the line 

" EventStoreDB can FIPS enabled OSes , this feature  requires a commercial licence " 
^ Term commercial licence need to be checked with @alexeyzimarev 

and leave the rest as is .


-

Reply:
I checked with Alexey, he's happy to call it 'commercial' here for now

Reply:
updated, thanks!